mathesar/templates/registration/login.html

```diff
@@ -1,26 +1,188 @@
     body {
-      padding: 2rem;
+      padding: 4rem 2rem;
+      background: var(--sand-200);
+      --login-page-spacing: var(--size-super-ultra-large);
+      color: var(--slate-800);
+      overflow: auto;
+      height: auto;
+    }
+    .align-center {
+      margin-left: auto;
+      margin-right: auto;
+    }
+    .logo img {
+      width: 13.25rem;
+      display: block;
+    }
+    .tutorial {
+      max-width: 48rem;
+      line-height: var(--size-xx-large);
+    }
+    .tutorial .body {
+      margin-top: var(--size-xx-small);
+    }
+    .login-card {
+      max-width: 28rem;
+      background-color: var(--white);
+      padding: var(--login-page-spacing);
+      border-radius: var(--border-radius-l);
+    }
+    .login-card h1 {
+      font-weight: 500;
+      font-size: var(--size-ultra-large);
+      margin: 0;
+      text-align: center;
+    }
+    .login-card .labeled-input + .labeled-input {
+      margin-top: var(--size-small);
+    }
+    .login-card .error-container {
+      margin-top: var(--size-small);
+      background: var(--red-100);
+      border: 1px solid var(--red-200);
+      border-radius: var(--border-radius-m);
+      padding: var(--size-x-small);
+    }
+    .login-card .labeled-input .help.error {
+      color: var(--red-600);
+    }
+    .login-card .footer {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+    .tutorial, .login-card,
+    .login-card form, .login-card .footer {
+      margin-top: var(--login-page-spacing);
     }
   </style>
 {% endblock %}
 
+{% block scripts %}
+  {% if development_mode %}
+    <script type="module" src="{{ client_dev_url }}/@vite/client"></script>
+    <script type="module" src="{{ client_dev_url }}/src/component-library/styles.scss"></script>
+    <script type="module" defer>
+      (function () {
+        document.body.style.visibility = 'visible';
+      })();
+    </script>
+  {% endif %}
+
+  <script>
+    function showStatusIndication() {
+      const loginButton = document.querySelector('#login_button');
+      loginButton.disabled = true;
+      loginButton.innerText = 'Logging In...';
+    }
+  </script>
+{% endblock %}
+
 {% block content %}
-  <h2>Log in to Mathesar</h2>
-  <form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <button type="submit">Log In</button>
-  </form>
+  <div class="logo">
+    <img src="{% static '/images/red-logo-with-text.svg' %}" alt="Mathesar Logo" class="align-center"/>
+  </div>
+
   {% if live_demo_mode %}
-    <p>
-      Mathesar is in live demo mode. Each session has its own copy of demo data.
-      Data will be deleted regularly.
-    </p>
+    <div class="tutorial align-center">
+      <div class="header">
+        <strong>Live Demo Mode</strong>
+      </div>
+      <div class="body">
+        Mathesar's live demo is available to anyone to try out. Use the following credentials to login:
+        <ul>
+          <li>username: <strong>demo</strong></li>
```

Comment:
I would specify the username and password here in Django settings instead of hardcoding.

Reply:
cc @mathemancer

Reply:
@kgodey I've added a commit https://github.com/centerofci/mathesar/pull/2191/commits/cf550691c8e0c7f06103367a478ec3050099d466 to handle this